ARM build failing and ARM python 3.9 missing wheels

The ARM build is failing due to an `error.h` error. The Python 3.9 wheels for ARM/aarch64 is missing as well.

> ARM builds are currently working fine on Travis. You're right on the missing ARM 3.9 build, cibuildwheel needs to be version bump.
> 
> Please at least provided the error messages, you're not giving me anything to work with here.

@TkTech Apologies, I typed this out late at night forgot to attach logs.

Below are the logs for the ARM build of pysimdjson 4.0:
```
root@afefd2207abe:/app# pip install pysimdjson -U
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/9617>
distutils: /usr/local/include/python3.9/UNKNOWN
sysconfig: /usr/local/include/python3.9
WARNING: Additional context:
user = False
home = None
root = None
prefix = None
Requirement already satisfied: pysimdjson in /usr/local/lib/python3.9/site-packages (3.2.0)
Collecting pysimdjson
  Downloading pysimdjson-4.0.0.tar.gz (453 kB)
     |████████████████████████████████| 453 kB 6.0 MB/s
Building wheels for collected packages: pysimdjson
  Building wheel for pysimdjson (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /usr/local/bin/python -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-0s_jf602/pysimdjson_b77bf0233f1f4d0d97bd75f49c07559f/setup.py'"'"'; __file__='"'"'/tmp/pip-install-0s_jf602/pysimdjson_b77bf0233f1f4d0d97bd75f49c07559f/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-7vtpwf6a
       cwd: /tmp/pip-install-0s_jf602/pysimdjson_b77bf0233f1f4d0d97bd75f49c07559f/
  Complete output (19 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.linux-aarch64-3.9
  creating build/lib.linux-aarch64-3.9/simdjson
  copying simdjson/__init__.py -> build/lib.linux-aarch64-3.9/simdjson
  copying simdjson/__init__.pyi -> build/lib.linux-aarch64-3.9/simdjson
  copying simdjson/py.typed -> build/lib.linux-aarch64-3.9/simdjson
  running build_ext
  building 'csimdjson' extension
  creating build/temp.linux-aarch64-3.9
  creating build/temp.linux-aarch64-3.9/simdjson
  gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/local/include/python3.9 -c simdjson/csimdjson.cpp -o build/temp.linux-aarch64-3.9/simdjson/csimdjson.o
  simdjson/csimdjson.cpp:654:10: fatal error: errors.h: No such file or directory
   #include "errors.h"
            ^~~~~~~~~~
  compilation terminated.
  error: command '/usr/bin/gcc' failed with exit code 1
  ----------------------------------------
  ERROR: Failed building wheel for pysimdjson
  Running setup.py clean for pysimdjson
Failed to build pysimdjson
Installing collected packages: pysimdjson
  Attempting uninstall: pysimdjson
    Found existing installation: pysimdjson 3.2.0
    Uninstalling pysimdjson-3.2.0:
      Successfully uninstalled pysimdjson-3.2.0
    Running setup.py install for pysimdjson ... error
    ERROR: Command errored out with exit status 1:
     command: /usr/local/bin/python -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-0s_jf602/pysimdjson_b77bf0233f1f4d0d97bd75f49c07559f/setup.py'"'"'; __file__='"'"'/tmp/pip-install-0s_jf602/pysimdjson_b77bf0233f1f4d0d97bd75f49c07559f/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-oa8s9me8/install-record.txt --single-version-externally-managed --compile --install-headers /usr/local/include/python3.9/pysimdjson
         cwd: /tmp/pip-install-0s_jf602/pysimdjson_b77bf0233f1f4d0d97bd75f49c07559f/
    Complete output (19 lines):
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-aarch64-3.9
    creating build/lib.linux-aarch64-3.9/simdjson
    copying simdjson/__init__.py -> build/lib.linux-aarch64-3.9/simdjson
    copying simdjson/__init__.pyi -> build/lib.linux-aarch64-3.9/simdjson
    copying simdjson/py.typed -> build/lib.linux-aarch64-3.9/simdjson
    running build_ext
    building 'csimdjson' extension
    creating build/temp.linux-aarch64-3.9
    creating build/temp.linux-aarch64-3.9/simdjson
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/local/include/python3.9 -c simdjson/csimdjson.cpp -o build/temp.linux-aarch64-3.9/simdjson/csimdjson.o
    simdjson/csimdjson.cpp:654:10: fatal error: errors.h: No such file or directory
     #include "errors.h"
              ^~~~~~~~~~
    compilation terminated.
    error: command '/usr/bin/gcc' failed with exit code 1
    ----------------------------------------
  Rolling back uninstall of pysimdjson
  Moving to /usr/local/lib/python3.9/site-packages/csimdjson.cpython-39-aarch64-linux-gnu.so
   from /tmp/pip-uninstall-8obiz9cn/csimdjson.cpython-39-aarch64-linux-gnu.so
  Moving to /usr/local/lib/python3.9/site-packages/pysimdjson-3.2.0.dist-info/
   from /usr/local/lib/python3.9/site-packages/~ysimdjson-3.2.0.dist-info
  Moving to /usr/local/lib/python3.9/site-packages/simdjson/
   from /usr/local/lib/python3.9/site-packages/~imdjson
ERROR: Command errored out with exit status 1: /usr/local/bin/python -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-0s_jf602/pysimdjson_b77bf0233f1f4d0d97bd75f49c07559f/setup.py'"'"'; __file__='"'"'/tmp/pip-install-0s_jf602/pysimdjson_b77bf0233f1f4d0d97bd75f49c07559f/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-oa8s9me8/install-record.txt --single-version-externally-managed --compile --install-headers /usr/local/include/python3.9/pysimdjson Check the logs for full command output.
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -15,7 +15,7 @@ env:
   - secure: 0pA9waJ8KLstYc9LdrJmUnQB2eqxCnEqBjZMWSU3t+NNfVyLVq+9+6aMFEbPtOD0LYSWFSAMOlyLvBmjaEzl82fu6yuJdkFmT9bYt3sBv9w+GnkHgQi7eom2gIWwcWx5PaSYx2CDrYBjCwBipK87tUoQCj+DuevqmunXSHDwVxf05k2KJB+23FlAtJDmbQi2VRS2shrMchemftOpbspyjS6sc9RZBWj/RhUyISeuDi+2yk+2HRGhbGJ0G+S54pQFIlLc04E+QcpLKnH8qUWmf8zf+La5WAtOPLDCBOmbWM8UCJfDg4UZ2XQH2z6Er82DfY4oz+nU7qoKoFKJ7OTTgwI1yvsuXa/aYK2wqPQUGNv2kKA6zZOxlmMxFJumO0Ctri76LwyG+qD4aU3Ca+05tfedswAcatr9MXN2LLWLLSjX8rSIW8hQmQ2PItP68ZXWs5oxFCbpbD9Z/Wu1tD3m0eMDwoAWfUBeX8Zdxo4bBG8WzG9vJWDJo1Lqgjz/zibtf1PefCCdyxBsY/jTgxl2gtwbSqz50+7+2jB3KuBaU08uC/aq4BLQu1qdkrPYkQQDjTgcRIrM1gn9cVuLUoGnFwcm8RHhEkY0IU529iIn0So0YrGvEFUPlEU/x9WlbLfWLms9UnmTqkeR4E3YnOTpP8Avb9PWL6G62cTukwPN2ic=
 
 install:
-- python3 -m pip install cibuildwheel==1.5.2
+- python3 -m pip install cibuildwheel==1.11.1.post1
 
 script:
 - python3 -m cibuildwheel --output-dir wheelhouse

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,3 +1,4 @@
 include simdjson/binding.cpp
 include simdjson/simdjson.h
 include simdjson/simdjson.cpp
+include simdjson/errors.h

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -2,3 +2,4 @@ include simdjson/binding.cpp
 include simdjson/simdjson.h
 include simdjson/simdjson.cpp
 include simdjson/errors.h
+include simdjson/errors.cpp

--- a/setup.py
+++ b/setup.py
@@ -78,7 +78,7 @@ else:
 setup(
     name='pysimdjson',
     packages=find_packages(),
-    version='4.0.0',
+    version='4.0.1',
     description='simdjson bindings for python',
     long_description=long_description,
     long_description_content_type='text/markdown',

